pages/common/xml-unescape.md

```diff
@@ -0,0 +1,16 @@
+# xml-unescape
+
+> Un-escape special XML characters.
+> More information: <http://xmlstar.sourceforge.net/docs.php>.
+
+- Unescape special XML characters from a string:
+
+`xml unescape {{"&lt;a1&gt;"}}`
+
+- Unescape special XML characters from standard input:
+
+`echo  {{"&lt;a1&gt;"}} | xml unescape`
```

Comment:
The `{{token_syntax}}` should probably go on the inside of the quotes here - same with the previous example too.
```suggestion
`echo  "{{&lt;a1&gt;}}" | xml unescape`
```

Reply:
Changed all places.

Reply:
xml-escape.md:`echo  "{{<a1>}}" | xml escape`
xml-unescape.md:`echo  "{{&lt;a1&gt;}}" | xml unescape`